unity_outline.md

```diff
@@ -0,0 +1,112 @@
+
+4. Adding Players to your Game
+    - Adding Player 1
+        - Adding the Sprite and Mapping the Components to the Sprite: In this checkpoint we'll show the student how to add components such as the box collider to a Sprite and explain why we need to do that.
+        - Setting Player1 position in GameManager.cs
+    - Adding Player 2
+        - Assignment: Setting Player2 position in GameManager.cs
+
+5. Adding the Ball to your Game
+    - Adding the Sprite
+    - How do we make the ball bounce?  There are three components that help define the bounce of the ball, we have seen two of those components used in the past and we will give you the third component.  First, take a stab at adding the two components to the ball sprite that you think would define it's bounce properties.  The third component will be a 2D physics material we define the actual bounciness property on and apply to an aforementioned component. 
+        - Assignment: have the student create both the 2D circle collider and a 2D rigid body
+        - Adding a 2D physics material
+        - Adding a ball script
+            - Breakpoint: more programming concepts
+                - Introduction to random numbers in CSharp to control start of ball motion.
+                - Introduction to debug logging.
+                - Introduction to the console (purposefully have them script an error so we can see it in the console and debug it)
+            - Add spin onto the ball
+                - Adding mass to the players so that they aren't pushed
+                - Show student how to do vector math to make the ball have some spin in the Ball script
+
+6. What is a game without a score?  In this section we learn how to display the score in Pong
+    - Create the ScoreManager script
+    - Add ScoreManager script to GameManager
+    - Set left and right wall triggers and create script that attaches to ScoreManager named 'SideWalls'
+    - Add the Score skin which is basically a label
+    - Set the font on the score by adding a custom asset
+
+7. We now have a functional game, but we can only play through one point.  In this checkpoint you will learn how to reset the ball's position when a point is scored.
+    - More scripting in the walls management script to send a signal to the ball script to tell it to reset.
+    - More scripting in the ball script to reset it's position when a point is scored
+
+8. Our game is cool but we don't have sound, how do we add sound to our game?
+    - Add audtio source to ball strike
+    - Adding pitch variation to the ball strike
+    - Assignment: now you add a sound when the player scores
+    - Lastly, let's add background music to the game
+        - Breakpoint: Audio optimization
+            - Since the background track is larger in size and longer we will want to make sure that the settings are correct such as:
+                - 'Use compression'
+                - 'Stream from disc'
+
+9. Miscellaneous Stuff, tweaking our game
```

Comment:
Some of these topics seem to lend themselves to modularity... just thinking out loud.